### DISCUSSION TITLE
"Update" feature for dot releases

### DISCUSSION THREAD

**Discussion by cgdive-todor** [Victor-IX/Blender-Launcher-V2#77] [Ideas]
I often find myself "updating" to newer releases by deleting the current installation and downloading the newest one. This is especially common with LTS installs. 
It would be nice to have this automated. 
Something like this:
![bl_update](https://github.com/Victor-IX/Blender-Launcher-V2/assets/48702545/de4a8a24-7680-4d70-ad09-fb66d242d2b9)

**Comment by cgdive-todor** [comment]
I guess the proposal in the mocap is a bit heavy-handed. It could be an option in the right-click menu and it probably has to display a warning that custom configs will be deleted.

**Comment by Victor-IX** [comment]
I like this idea! As you said in your second message a right-click option could be a good option.
I will create a feature request for that

**Reply by cgdive-todor** [reply]
Great, thanks for considering the idea!
